Libraries/xcexecution/Sources/SimpleExecutor.cpp

```diff
@@ -258,7 +257,8 @@ performInvocations(
                         processContext->groupID(),
                         processContext->userName(),
                         processContext->groupName());
-                    success = driver->run(&context, filesystem) != 0;
+                    int exitCode = driver->run(&context, filesystem);
+                    success = (exitCode == 0);
```

Comment:
Amazing.  Not really a fan of adding 'int exitCode' since it's used in this one case, but I don't care enough either way.